src/bootstrap/src/core/build_steps/test.rs

```diff
@@ -3266,6 +3267,7 @@ impl Step for CodegenGCC {
                 SourceType::InTree,
                 target,
                 "run",
+                false,
```

Comment:
Random unexplained booleans make code hard to read. This should either be an enum with meaningful variant names, or it needs a comment explaining what this `false` means.